openshift/e2e-tests-openshift.sh

```diff
@@ -34,8 +34,7 @@ function install_tekton_pipeline() {
 }
 
 function create_pipeline() {
-  resolve_resources config/ tekton-pipeline-resolved.yaml "nothing" $OPENSHIFT_REGISTRY/$OPENSHIFT_BUILD_NAMESPACE/stable
```

Comment:
are you removing the resolve_resources function ? then please remove it completely, or are we going to have  two different functions doing slighty much the same?